formlibrary/serializers.py

```diff
@@ -45,4 +49,32 @@ class HouseholdListDataSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Household
-        fields = ['id', 'name', 'program', 'created_by', 'create_date']
+        fields = ['id', 'name', 'duration', 'program', 'created_by', 'create_date']
+
+
+class TrainingListDataSerializer(serializers.ModelSerializer):
+    program = ProgramSerializer()
+    created_by = ActivityUserSerializer()
+
+    class Meta:
+        model = Training
+        fields = ['id', 'name', 'created_by', 'create_date', 'program']
+
+
+class DistributionSerializer(serializers.ModelSerializer):
```

Comment:
Before we merge it, I saw the same code in other Pr, is it by mistake?

Reply:
let me confirm where there is another occurrence. but its probably a mistake on my part